content/about/faq/_index.md

```diff
@@ -12,24 +12,24 @@ A: Nah.
 
 ### Q: What Language/Engine is SS14 written in?
 
-A: It's written in C# using a hand made engine. C# is a good language and it's pretty much perfect for the project. It's quite fast but also very comfortable to write.
+A: It's written in C# using a hand made engine, RobustToolbox. C# is a good language and it's pretty much perfect for the project. It's quite fast but also very comfortable to write.
 
 ### Q: Can I Help?
 
-A: Sure you can. Hop on Discord or IRC and look at GitHub. We can use any help we can get!
+A: Sure you can. Hop on Discord or IRC and look at GitHub and our [HackMD Wiki](https://hackmd.io/@ss14/docs/%2FvJp0-PyuSNeWMb5GA00B4Q). We love new coders and will always answer questions and provide coding help!
 
-### Q: Will SS14 Support Multiple Codebases?
+### Q: Are you Unitystation?
 
-A: It already does!
+A: No. Unitystation is a completely different project with it's own design decisions, goals, and runs on an entirely different engine.
 
-### Q: hdisry?
+### Q: Will SS14 Support Multiple Codebases?
 
-A: No.
+A: It already does! If you're looking to start your own SS14 codebase fork the [content repo](https://github.com/space-wizards/space-station-14) and read the [getting started](https://hackmd.io/@ss14/docs/%2FBZkI4RlUQbm09QWrXCZ3kg) guide.
 
 ### Q: Is There a Patreon? Can I donate?
 
-A: No. We have no methods of donating to the project at this time. We don't think it'd help to speed up development. While a system like bounties *might* be nice to encourage contribution, it's also a hassle and a potential source of drama.
+A: [Yes.](https://github.com/space-wizards/space-station-14) We started a Patreon in order to pay for productivity software, server hosting, work commissions, and more! In addition to providing rewards, we will also regularly post a financial statement that shows how our income is being used in the game's development. We want to give our patrons full trust and confidence, and we believe that being 100% transparent about how the money is being used is best.
 
-### Q: Will SS14 Go On Steam?
+### Q: Where can I play SS14?
 
-A: Yes! We're actively working on getting the store page ready for publishing. It'll obviously be free on steam too.
+A: You can download SS14 right now from our website, or wait for our free Steam release which we estimate will come out in Fall of 2020.
```

Comment:
You had one job man.

Reply:
What's wrong with this

Reply:
You could at least mention that the game isn't playable yet. i.e. only for testing.